commitizen/commands/changelog.py

```diff
@@ -106,6 +114,11 @@ def write_changelog(
                 changelog_out = changelog_hook(changelog_out, partial_changelog)
             changelog_file.write(changelog_out)
 
+    def export_template(self):
```

Comment:
Just curious. Is this function mainly for users to debug their template? or when do we expect the users to use it?

Reply:
This addition was following https://github.com/commitizen-tools/commitizen/pull/645#issuecomment-1374407611. The idea is to provide users and plugin developpers with:
- a way to inspect the template
- a way to quickly customize it locally (export to current dir, do the change you want)